packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -342,6 +349,33 @@ const ComboBox = forwardRef(
       slug,
       ...rest
     } = props;
+    const { refs, floatingStyles } = useFloating(
+      autoAlign
+        ? {
+            strategy: 'fixed',
+            middleware: [
+              flip({
+                fallbackAxisSideDirection: 'none',
+              }),
```

Comment:
`none` is the default so we can remove it

```suggestion
              flip(),
```